src/main/java/com/diffblue/deeptestutils/Reflector.java

```diff
@@ -95,7 +95,7 @@ public static void setField(final Object obj, final String fieldName,
    *     subclass or implementor thereof), or if an unwrapping conversion fails.
    * @exception IllegalAccessException if an error occurs
    */
-  private static <T> void setField(final Class<T> c, final Object o,
+  public static <T> void setField(final Class<T> c, final Object o,
```

Comment:
This used to work (I recall it being the first bug you fixed!) - how come we now need this method?